script/start.ts

```diff
@@ -0,0 +1,72 @@
+import * as express from 'express'
+import * as webpack from 'webpack'
+import * as devMiddleware from 'webpack-dev-middleware'
+import * as hotMiddleware from 'webpack-hot-middleware'
+
+import { forceUnwrap as u } from '../app/src/lib/fatal-error'
+
+import configs = require('../app/webpack.development')
```

Comment:
❓ Why do we use `require` here?

Reply:
I’m using `import =` since that file uses `export =`.

Reply:
![](https://user-images.githubusercontent.com/359239/42241787-272a63dc-7ee2-11e8-9e83-573b0b995737.png)

I think the shape of the webpack config files doesn't really give us much to work with here

Reply:
I remember a little better now — We could use `import * as configs`, but the file exports an array, but when you use `import *`, you get an object.